client/state/posts/reducer.js

```diff
@@ -282,6 +281,23 @@ export function edits( state = {}, action ) {
 				[ action.siteId ]: omit( state[ action.siteId ], action.postId || '' )
 			} );
 
+		case POST_SAVE_SUCCESS:
+			if ( ! state.hasOwnProperty( action.siteId ) || ! action.savedPost ) {
+				break;
+			}
+			const { postId, siteId, savedPost } = action;
+
+			if ( postId ) {
+				return state;
```

Comment:
Should we be consistent with how we exit early from the switch, between this and the `break;` above which achieve the same goal?